sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -348,6 +348,7 @@ abstract class HiveComparisonTest
                 queryString.replace("../../data", testDataPath))
               val containsCommands = originalQuery.analyzed.collectFirst {
                 case _: Command => ()
+                case _: InsertIntoTable => ()
```

Comment:
ah, it's because I removed the `PreInsertionCasts` rule, which turns `InsertIntoTable` to `InsertIntoHiveTable`. This conversion doesn't matter, as hive planner will plan `InsertIntoTable` into physical `InsertIntoHiveTable`.

So adding a case here is a reasonable fix.